### DISCUSSION TITLE
`percy upload -v .` fails uploading base images.

### DISCUSSION THREAD

**Discussion by 50kudos** [percy/cli#1222] [Support] [answered]
## The problem
Failed uploading images the first time with `percy upload` command.

## Environment

- Node version: 17.9.1
- `@percy/cli` version: 1.21.0
- Version of Percy SDK you’re using: (*upload images directly*)
- OS version: macOS 11.6.1 (20G224)
- Type of shell command-line [interface]: Bash

## Details
It doesn't seem to be finalizing "comparison" before finalizing the build.

## Debug logs

![Screen Shot 2023-03-31 at 7 37 13 AM](https://user-images.githubusercontent.com/2753398/229003386-546ff17d-fc6e-41fc-8db5-489770beb294.jpg)

## Code to reproduce issue

`PERCY_BRANCH=percy-setup PERCY_TOKEN=secret percy upload  .`

**Comment by itsjwala** [comment, marked as answer]
@50kudos 

* From what I can see, you're trying to create baselines for App Percy(Native apps vrt) builds with `percy upload` command 
  *  If this is the case, its not possible currently, There is a workaround though please reach out to our support team they'll be able to assist you further. 
* If this is Web project, then you're using Apps projects `PERCY_TOKEN` you'll need to grab web projects tokens to run `percy upload`
